### ISSUE TITLE
TS error: Module '"@floating-ui/dom"' has no exported member 'Side'

### ISSUE THREAD

**Issue by WickyNilliams** [floating-ui/floating-ui#1605]
This worked in earlier versions, but no longer seems to work:

```ts
import { Side } from "@floating-ui/dom";
```

I can't see anything obvious in the commit history that will have caused this. The other type exports don't seem to be discoverable by TS either. I wonder whether this is caused by not having an explicity `types` field in the `package.json`?

Codesandbox: https://codesandbox.io/s/reverent-tdd-p0gz97?file=/src/index.ts

**Comment by atomiks** [comment]
Nope, for some reason the `src` folder is no longer being generated in `0.4.3` :\ https://unpkg.com/browse/@floating-ui/dom@0.4.3/

**Comment by WickyNilliams** [comment]
Ah, good spot!

Still, I think it's recommended to be explicit with a types field? See: https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package

**Comment by atomiks** [comment]
True, it also shows it on npm, but @FezVrasta prefers "types" to be agnostic because Flow exists 😄
